devel/libupnp/Portfile

```diff
@@ -4,28 +4,22 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
 
-github.setup        pupnp pupnp 1.14.12 release-
-
-github.tarball_from releases
```

Comment:
please keep using "releases"; you'll also need to put "use_bzip yess" back. And do not make unnecessary whitespace changes for maintained ports. Minimal changes please to do the update.

Reply:
The releases tarball is broken, I have pointed out at that in comments.

Reply:
I'll take a look as that sounds strange... in the meantime please revert unneeded formatting changes (i.e., whitespace or reordering of lines)